packages/terra-framework-docs/src/terra-dev-site/test/collapsible-menu-view/CollapsibleMenuMultiSelect.test.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import IconTable from 'terra-icon/lib/icon/IconTable';
+import IconFlowsheet from 'terra-icon/lib/icon/IconFlowsheet';
+import IconVisualization from 'terra-icon/lib/icon/IconVisualization';
+import IconSend from 'terra-icon/lib/icon/IconSend';
```

Comment:
We should use the named exports introduced in https://github.com/cerner/terra-core/pull/3915.